examples/pytorch/graphsage/dist/train_dist_unsupervised.py

```diff
@@ -300,17 +165,13 @@ def run(args, device, data):
     # Unpack data
     train_eids, train_nids, in_feats, g, global_train_nid, global_valid_nid, global_test_nid, labels = data
     # Create sampler
-    sampler = NeighborSampler(g, [int(fanout) for fanout in args.fan_out.split(',')], train_nids,
-                              dgl.distributed.sample_neighbors, args.num_negs, args.remove_edge)
-
-    # Create PyTorch DataLoader for constructing blocks
-    dataloader = dgl.distributed.DistDataLoader(
-        dataset=train_eids.numpy(),
-        batch_size=args.batch_size,
-        collate_fn=sampler.sample_blocks,
-        shuffle=True,
-        drop_last=False)
-
+    neg_sampler = dgl.dataloading.negative_sampler.Uniform(args.num_negs)
+    sampler = dgl.dataloading.NeighborSampler([int(fanout) for fanout in args.fan_out.split(',')])
+    # Create dataloader
+    exclude = 'reverse_id' if args.remove_edge else None
+    reverse_eids = th.arange(g.num_edges()) if args.remove_edge else None
+    dataloader = dgl.dataloading.DistEdgeDataLoader(g, train_eids, sampler, negative_sampler=neg_sampler,
```

Comment:
Is `reverse_eids` a valid argument for `DistEdgeDataLoader`?

Reply:
Not, but it's supported  and parsed in https://github.com/dmlc/dgl/blob/cb5e3489c0694107295bc70804de1559e97cdb89/python/dgl/_dataloading/dataloader.py#L637.